apps/pushy/src/pushy_principal.erl

```diff
@@ -41,56 +41,74 @@ fetch_principal(OrgName, Requestor) ->
     end.
 
 -spec request_principal(OrgName :: binary(),
-                        Requestor :: binary()) -> #pushy_principal{}.
+                        Requestor :: binary()) -> [#pushy_principal{}].
 request_principal(OrgName, Requestor) ->
```

Comment:
Perhaps a rename of this function is in order now that we're requesting principals and always returning a list.